io.github.unknownskl.greenlight.yml

```diff
@@ -0,0 +1,79 @@
+app-id: io.github.unknownskl.greenlight
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+platform-extensions:
+  - org.freedesktop.Platform.VAAPI.Intel
```

Comment:
```suggestion
```

This is not a runtime.

Reply:
Thanks, first time building a flatpak. Will remove it